mdadm.h

```diff
@@ -408,8 +408,15 @@ struct mdinfo {
 	#define DS_BLOCKED	16
 	#define	DS_REMOVE	1024
 	#define	DS_UNBLOCK	2048
+	#define	DS_EXTERNAL_BB	4096
 	int prev_state, curr_state, next_state;
 
+	/* If set by monitor, managemon needs to remove faulty device */
+	bool man_disk_to_remove : 1;
```

Comment:
I think man_remove_disk would be sufficient, disk_to_remove sounds like you would specify a disk :D, but I's just me being nitpicky.

I'm fine either way.

Reply:
Because I'm specifying it. At least I'm saying that this is a disk and this filed is for managemon.

mdinfo in abused in mdadm. I added mon, to let reader know what it the purpose of this filed.

mdinfo could be array or disk now :(